deploys/reference-app-all-in-one.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: reference-app
+spec:
+  ports:
+  - port: 80
+    protocol: TCP
+    targetPort: 5000
+  selector:
+    run: reference-app
+  sessionAffinity: None
+  type: LoadBalancer
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: reference-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: reference-app
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        run: reference-app
+    spec:
+      containers:
+      - image: gcr.io/genuine-segment-148215/reference-app:v3
+        imagePullPolicy: IfNotPresent
+        name: reference-app
+        ports:
+        - containerPort: 5000
+          protocol: TCP
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+---
+apiVersion: v1
+kind: Pod
```

Comment:
I'm confused (and it may just be my poor understanding of Kubernetes, since I've only read the docs and haven't tried this yet): I thought the `Deployment.spec` above was _in place of_ defining a pod separately here. If you just replace the `Deployment.spec` with this `Pod.spec` (and remove the `Pod` entry entirely), does that accomplish what you're aiming for?